gk/main.c

```diff
@@ -745,6 +822,22 @@ gk_stage2(void *arg)
 	if (ret < 0)
 		goto cleanup;
 
+	if (ipv6_if_configured(&gk_conf->net->front)) {
+		int i;
+		for (i = 0; i < gk_conf->num_lcores; i++) {
+			unsigned int lcore = gk_conf->lcores[i];
+			ret = register_ipv6_acl(&gk_conf->net->front,
```

Comment:
The motivation to have the GK and GT blocks register is that they know which lcores, and therefore which sockets, need to have the ACL. What I'm seeing now is that the LLS block can be the only one that registers the ND rule, even if it installs it for sockets that will not use it. Is that correct?